src/pages/languages.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * @emails react-core
+ * @flow
+ */
+
+import Layout from 'components/Layout';
+import Container from 'components/Container';
+import Header from 'components/Header';
+import TitleAndMetaTags from 'components/TitleAndMetaTags';
+import React from 'react';
+import {media, sharedStyles} from 'theme';
+
+// $FlowFixMe This is a valid path
+import languages from '../../content/languages.yml';
+
+const sortedLanguages = languages.sort((a, b) =>
+  a.label.localeCompare(b.label),
+);
+
+type Props = {
+  location: Location,
+};
+
+const Languages = ({location}: Props) => (
+  <Layout location={location}>
+    <Container>
+      <div css={sharedStyles.articleLayout.container}>
+        <div css={sharedStyles.articleLayout.content}>
+          <Header>Languages</Header>
+          <TitleAndMetaTags title="React - Languages" />
+
+          <div css={sharedStyles.markdown}>
+            <p>
+              The React documentation is available in the following languages:
```

Comment:
Are going to list *all* the languages here or only the ones that are "completed"? I thought the idea was that you had to get some set of pages complete before the translation becomes "official." Languages like Tamil don't have any pages translated yet so it'll be disappointing to discover every page is still in English.

Reply:
The only one that's finished the "core pages" Dan and I defined is Spanish, (with Japanese and Brazilian Protuguese coming up soon). Maybe we could to something like 

## Translations

Spanish

Japanese

Brazilian Portuguese

## In Progress

The following languages are in progress. If you would like to contribute, please check the corresponding Github link.

Reply:
I'll defer to your recommendation here. We could add an attribute to the YML that specifies whether the language is complete enough to show or not. I wasn't sure what the plan was.

Reply:
My preference would be to show completed languages and link to isreacttranslatedyet.com for in-progress ones. @gaearon what do you think?

Reply:
I don't share this preference. I think surfacing in-progress translation on the official docs site will direct more energy and attention torward them.

Reply:
Hmm, I think you're right. Do you just want the languages page on the official site to replace isreacttranslatedyet.com?

Reply:
I don't think it's bad to have them both. I like the progress indicators on isreacttranslatedyet.com. Maybe if we add some sort of progress indicator to the official site then we could repurpose isreacttranslatedyet.com to just redirect, but that seems like something we could do as a follow up.